ipapython/dn.py

```diff
@@ -445,13 +445,11 @@ def _adjust_indices(start, end, length):
         end = length
     elif end < 0:
         end += length
-        if end < 0:
-            end = 0
+        end = max(end, 0)
```

Comment:
I find the old code more readable. It's more obvious was is happening here. It might even be faster (not that it matters).

Reply:
Logically it's the same. We can skip below all together:
```
consider-using-max-builtin (R1731)
Consider using '%s' instead of unnecessary if block Using the max builtin instead of a conditional improves readability and conciseness.

consider-using-min-builtin (R1730)
Consider using '%s' instead of unnecessary if block Using the min builtin instead of a conditional improves readability and conciseness.
```